app/views/agents/_form.html.haml

```diff
@@ -9,7 +9,7 @@
   = hidden_field_tag :deletable, deletable if deletable
   = hidden_field_tag agent_field_name(:id, name_prefix), agent.id if agent.id
 
-  - if is_organization?(agent) && params[:parent_id]
```

Comment:
why this line and also removing `params[:parent_id]`

Reply:
What's the purpose of this line?
- To show or not the radio selector to select (organization / person)
When do we want to hide it?
- Only when we try to create a user that is an affiliation (cause by default it's only an organization)

And the param params[:show_affiliations] is enough to determine that

Reply:
For this PR I'll just fix tests and then it's done

Reply:
I have no idea of the why of this line, but if you search in the code, you will a lot of references to this parent_id parameter. So, you can't just remove it without being sure of what it does and clean the references to it.

Reply:
I didn't remove the parent_id param, I'm just not depending on it in my if, I didn't change the value of it or something so anything else use it will not be affected

Reply:
no I mean it was used here for a purpose, that you need to understand before deciding to remove it or not.

Reply:
Ah yes, I have a deep understading in this ;)

Reply:
OK, I trust you on that.